full-stack-tests/backend/src/integration/AzuriteTest.ts

```diff
@@ -94,9 +94,10 @@ export namespace AzuriteTest {
         bcvDb.withSqliteStatement("SELECT v FROM kv WHERE k = 'writeLock'", (stmt) => {
           if (stmt.nextRow()) {
             const writeLockData = JSON.parse(stmt.getValueString(0));
-            const expiresDate = new Date(writeLockData.expires);
-            const currentTime = new Date();
-            resolve(expiresDate > currentTime);
+            const expiresTime = new Date(writeLockData.expires);
+            // should be 5 min but let's use 4.5min to avoid time conflict
+            const writeLockTimeRemains = 4.5*60*1000;
```

Comment:
How about changing 'isWriteLockRefreshed' to 'isWriteLockValidForAtleast' And taking writeLockTimeRemains as a parameter?

Reply:
Resolved in [commit 8a0fdb5818d3144a873d3ae42a56c9a99db05473](https://github.com/iTwin/itwinjs-core/pull/7000/commits/8a0fdb5818d3144a873d3ae42a56c9a99db05473)